examples/app_with_observability/app_async.py

```diff
@@ -38,7 +38,7 @@ async def llm_call(prompt):
     }
 
 
-@ag.entrypoint()
+@ag.entrypoint
```

Comment:
```suggestion
@ag.entrypoint()
```

Tracing won't work if you don't call the decorator class.

Reply:
Try interacting with it on the playground using the latest version available at `cloud.agenta.ai` within the team's workspace. After using the app from the playground, check the traces view. You'll find that no traces were generated.

Reply:
The issue is that we are initializing ag.init with wrong data, not because of the entrypoint. 

The order I made for the initializaiton of the vars are:
1. Explicit through ag.init()
2. explicit though ag.init config_fname
3. through env vars

Do you think it makes sense? (do you have analogies in other libs we can compare to?)

Reply:
Ah, I get it now. Yes, it makes sense. Thanks for explaining.

Edit: I saw the observability documentation. It resolved my concern. ;-)